mcr.cpp

```diff
@@ -29,20 +29,17 @@ int main(){
 				cout << "Player 1: ";
 			else
 				cout << "Player 2: ";
-			cout << "Which cell to mark? i:[1..3], j:[1..3]: ";
+			cout << "Which cell to mark? i:[0..2], j:[0..2]: ";
 			cin >> i >> j;
```

Comment:
Check if mark is legal

Reply:
You're right. should check if the cell is marked or not